sdk/loadtestservice/azure-developer-loadtesting/setup.py

```diff
@@ -38,7 +38,7 @@
     url="https://github.com/Azure/azure-sdk-for-python/tree/main/sdk",
     keywords="azure, azure sdk",
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
```

Comment:
please update the code with https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-with-tools#update and codegen will help to update `setup.py` automatically

Reply:
pushed code after running the scripts